CONTRIBUTING.md

```diff
@@ -1,5 +1,11 @@
 # Contributing to Node.js
 
+## Code of Conduct
+
+The Code of Conduct explains the *bare minimum* behavior
+expectations the Node Foundation requires of its contributors. 
+[Please read it before participating.](./CODE_OF_CONDUCT.md)
```

Comment:
My personal choice of wording would be removing "before participating" here, similar to what you do below.